### ISSUE TITLE
Touch Bar cannot get the correct video length on some websites

### ISSUE THREAD

**Issue by hellomduo** [AdguardTeam/AdGuardForSafari#133]
Touch Bar cannot get the correct video length on some websites when the adguard is open ("block on this website" is checked), it only shows the video buffered time.

### Steps to reproduce

1. example: https://v.qq.com/x/cover/jcezu7ev565timo/e00235zk2z7.html
2. when adguard is closed , touchbar shows
<img width="1085" alt="2019-02-16 15 45 11" src="https://user-images.githubusercontent.com/37796453/52896374-e9c70080-3201-11e9-8d5d-21184d4a6034.png">

3. when adguard is open , touchbar shows 
<img width="1085" alt="2019-02-16 15 44 48" src="https://user-images.githubusercontent.com/37796453/52896380-fa777680-3201-11e9-840e-92e7ae439c22.png">

4. the correct video length is about 1h 27min. (Unmute will show Touch Bar)

### Expected behavior

touchbar shows the correct video length when you watch video.

### Actual behavior

touchbar shows the buffered time.

<details><summary>Screenshots:</summary>
 plesas see STEPS TO REPRODUCE

</details>

### Your environment

* AdGuard for Safari version: Mac App Store version (The latest version)
* Environment name and version: Safari 12.0.3 (14606.4.5)
* Operating System: macOS Mojave 10.14.3 (18D109)
* Filters you use in AdGuard:
<img width="951" alt="2019-02-16 15 59 38" src="https://user-images.githubusercontent.com/37796453/52896499-e9c80000-3203-11e9-9456-7554fde85ab4.png">

* Any other browser extensions you have installed: evernote web clipper / translate / search engine switcher

**Comment by ameshkov** [comment]
Well, this issue is really weird. It might be caused by the filters, though, not by AdGuard itself.

Are there any other examples besides qq.com?

**Comment by zzebrum** [comment]
@hellomduo disable all filters and try to reproduce the issue, please.

**Comment by hellomduo** [comment]
@zzebrum @ameshkov  yes, i turn off all filters and try again, the issue will not happen. It caused by the filters, thanks for your help！

**Comment by iDjay** [comment]
having the same issue on Twitch!

**Comment by Mizzick** [comment]
@Alex-302 pls have a look?

**Comment by Alex-302** [comment]
![image](https://user-images.githubusercontent.com/8361299/54830819-e8399e00-4cc1-11e9-8f8f-6135a60a5b27.png)

**Comment by iDjay** [comment]
@Alex-302 did you try twitch?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by hellomduo** [comment]
i try twitch has same issue.
for example: https://www.twitch.tv/videos/398257896
touchbar shows
<img width="1085" alt="触控栏快照 2019-03-22 23 08 30" src="https://user-images.githubusercontent.com/37796453/54832623-7aa86480-4cf7-11e9-8157-e827995b1f91.png">

**Comment by iDjay** [comment]
https://www.twitch.tv/videos/399116225 here‘s an example 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Alex-302** [comment]
@iDjay I have no device with that touch bar.

**Comment by iDjay** [comment]
@Alex-302 so you won‘t be able to fix it?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Alex-302** [comment]
I can check it using teamviewer

**Comment by iDjay** [comment]
You can actually emulate the Touch Bar with apps for Mac / iPad

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Alex-302** [comment]
I emulate even MacOS:)

**Comment by iDjay** [comment]
Wow I just realized that it‘s an issue with twitch and not with AdGuard 🤦🏼‍♂️ for some reason it doesn‘t show the correct length when AdGuard is disabled too. Still not sure about the first link from @hellomduo 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
